templates/Email/EmailThread.html

```diff
@@ -1,4 +1,45 @@
-<div class="coveo-result-frame coveo-email-result">
+<div class="coveo-result-frame">
+  <div class="coveo-result-cell" style="vertical-align:top;text-align:center;width:32px;">
+    <span class="CoveoIcon" data-small="true" data-with-label="false"></span>
+    <div class="CoveoQuickview" data-template-id="EmailQuickviewContent"></div>
+  </div>
+  <div class="coveo-result-cell" style="vertical-align: top;padding-left: 16px;">
+    <div class="coveo-result-row" style="margin-top:0;">
+      <div class="coveo-result-cell coveo-no-wrap" style="vertical-align:top;font-size:16px;">
+        <a class="CoveoResultLink"></a>
+      </div>
+      <div class="coveo-result-cell" style="width:120px;text-align:right;font-size:12px">
+        <div class="coveo-result-row">
+          <span class="CoveoFieldValue" data-field="@date" data-helper="date"></span>
+        </div>
+      </div>
+    </div>
+    <div class="coveo-result-row" style="margin-top:12px;">
+      <div class="coveo-result-cell">
+        <span class="CoveoFieldValue" data-field="@from" data-helper="email" data-html-value="true" data-text-caption="From:" style="margin-right:30px;"></span>
+        <span class="CoveoFieldValue" data-field="@recipients" data-helper="email" data-html-value="true" data-text-caption="To:" style="margin-right:30px;"></span>
+      </div>
+    </div>
+    <div class="coveo-result-row" style="margin-top:12px;">
+      <div class="coveo-result-cell">
+        <span class="CoveoExcerpt"></span>
+      </div>
+    </div>
+    <div class="coveo-result-row" style="margin-top:12px;">
+      <div class="coveo-result-cell" style="padding-top:5px; padding-bottom:5px; font-size:13px;">
+        <span class="CoveoResultAttachments" data-result-template-id="EmailResultAttachment">
+        </span>
+      </div>
+    </div>
+    <div class="coveo-result-row" style="margin-top:12px;">
+      <div class="coveo-result-cell" style="padding-top:5px; padding-bottom:5px; font-size:13px;">
+        <span class="CoveoResultFolding" data-result-template-id="EmailChildResult" data-more-caption="ShowAllReplies" data-less-caption="ShowOnlyMostRelevantReplies"></span>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!--<div class="coveo-result-frame coveo-email-result">
```

Comment:
Yes those were the old templates, was keeping them for reference but I'll check history instead.